overlays/default.nix

```diff
@@ -113,8 +113,6 @@ in
 
   beautyline-icons = final.callPackage ../pkgs/beautyline-icons { };
 
-  blurredwallpaper = final.callPackage ../pkgs/blurredwallpaper { };
```

Comment:
Can we make this an alias of `plasma-plugin-blurredwallpaper`? That will retain backwards compatibility.

Reply:
I'm not sure how to exactly do that, but sounds like a good idea. Eventually with a deprecation notice? 👀

Reply:
I usually do 
```
blurredwallpaper = pkgs.plasma-plugin-blurredwallpaper;
```
but CC @PedroHLC if that's the correct way to do it here.